### DISCUSSION TITLE
Can TreeView support using up/down arrow to select tree node?

### DISCUSSION THREAD

**Discussion by benny856694** [bdlukaa/fluent_ui#797] [General]
it's would be more accessible if up/down arrow key can be supported

**Comment by bdlukaa** [comment]
Hello! They are part of the Flutter focus system. They work on Windows, but I'm afraid they don't work on the web (platform behavior).

**Reply by benny856694** [reply]
No, doesn't work on Windows either. My project is for Windows, and it doesn't work

**Reply by bdlukaa** [reply]
That is odd. Can you show us a GIF of the behavior you get?

**Comment by benny856694** [comment]
Here it is: [screen shot](https://1drv.ms/i/s!Al4RslLhazbZozKPBWrWpQiUKX8H?e=Kdt3eO), when I press up/down, the focus moves to the right widget.

![image](https://user-images.githubusercontent.com/45696119/230151494-bb1e0f5b-86e0-4347-9dca-271dfb2fcdd0.png)

What happens when you press Tab or Shift-Tab? The same behavior?

**Reply by benny856694** [reply]
yes, same behavior

**Reply by benny856694** [reply]
I simplified the widget tree by removing the parent widget of the TreeView, now the up/down key works

**Reply by bdlukaa** [reply]
You can attempt to give focus to the entire tree view using a `FocusScope` and `autofocus: true`

**Reply by benny856694** [reply]
up/down arrow still not working by wrapping TreeView inside a FocusScope and setting autofocus to true
